docs/document/docs/guides/table-readonly-table-customization.md

```diff
@@ -0,0 +1,514 @@
+
+![render](assets/table-edit-text.gif)
+When you edit the input in the first cell, the Text component will trigger `change` event with `detail.value = "user1 edited"`.
+The table then catch that value and assigns it back to the `username` field.
+Now the data of the table will be:
+```javascript
+[{ username: "user1 edited" }, { username: "user2" }];
+```
+
+### Modify the event detail
+If you want to customize the value the table receives then you need to modify it in the `change` event before the table receives it.
+We also use the sample code above and add the following content to `renderName` function:
+
+```javascript
+// table.js
+...
+const renderName = (cellData) => {
+  const text = new Text({ value: cellData });
+
+  // Modify the value before it bubble to table cell
+  text.addEventListener('change', (event) => {
+    event.detail.value = "modified value" // add any value you want set to username;
+  });
+  return text;
+};
+...
+```
+When you edit the input in the first cell, The value received by the table will always be `"modified value"`
+
+## Examples
+### Basic Usage
+
+#### ReadOnlyTable
+![readonly-table](assets/readonly-table.png)
+
+Display a read-only mode table.
+<details>
+  <summary>Show code</summary>
+
+  ```js
+const columns = [
+    {
+        title: 'Name',
+        field: 'name',
+    },
+    {
+        title: 'Gender',
+        field: 'gender',
+    },
+    {
+        title: 'Address',
+        field: 'address',
+    },
+];
+
+const data = [
+    {
+        name: 'John Brown',
+        gender: 'male',
+        address: 'osaka-japan',
+    },
+    {
+        name: 'Jim Green',
+        gender: 'female',
+        address: 'tokyo-japan',
+    },
+    {
+        name: 'Joe Black',
+        gender: 'male',
+        address: 'hochiminh-vietnam',
+    },
+];
+
+const readOnlyTable = new Kuc.ReadOnlyTable({columns, data});
+const space = kintone.app.record.getSpaceElement('space');
+space.appendChild(readOnlyTable);
+  ```
+</details>
+
+#### Table
+![table](assets/table.png)
+
+You can manipulate with the components displayed in the table.</br>
+Use the `columns.render` function to specify the component to be displayed in the cell. Please check `renderGender` and `renderAddress` functions in below code example:
+<details>
+  <summary>Show code</summary>
+
+  ```js
+// render gender column with dropdown
+const renderGender = (cellData) => {
+    const radioButton = new Kuc.RadioButton({
+        items: [
+        {
+            label: 'Male',
+            value: 'male',
+        },
+        {
+            label: 'Female',
+            value: 'female',
+        },
+        ],
+        itemLayout: 'vertical',
+        value: cellData,
+    });
+
+    return radioButton;
+};
+
+// render address column with dropdown
+const renderAddress = (cellData) => {
+    const country = cellData.split('-')[1];
+    const dropdownCountry = new Kuc.Dropdown({
+        items: [
+        {
+            label: 'Việt Nam',
```

Comment:
Please use English name. Please update image, too.
```suggestion
            label: 'Viet Nam',
```

Reply:
I updated the image